config/config.sample.php

```diff
@@ -578,6 +578,42 @@
  */
 'trashbin_purge_limit' => 50,
 
+/**
+ * Define trashbin folder skip list
+ * Define a list of folders that will skip the trashbin and therefore be deleted immediately
+ * The setting expects folder names with or without trailing slash. All the content
+ * inside such folders will also skip the trashbin.
+ *
+ * WARNING: THIS SETTING IS SYSTEM WIDE AND COULD CAUSE DATA LOSS. USE THIS ONLY IF YOU KNOW WHAT YOU ARE DOING.
+ */
+'trashbin_skip_directories' => [
+	'temp',
+],
+
+/**
+ * Define trashbin file extension skip list
+ * Define a list of file extensions to determine files that will skip the trashbin and therefore be deleted immediately
+ * Values must not have a leading ".", otherwise corresponding files wont be detected.
+ *
+ * WARNING: THIS SETTING IS SYSTEM WIDE AND COULD CAUSE DATA LOSS. USE THIS ONLY IF YOU KNOW WHAT YOU ARE DOING.
+ */
+'trashbin_skip_extensions' => [
+	'iso',
+	'mkv',
+],
+
+/**
+ * Define trashbin threshold size
+ * Define a threshold for files to skip the trashbin and delete immediately
+ * Once the size of a resource is greater than or equal the given value, the trashbin will be skipped.
+ *
+ * All positive numbers and zero are allowed. Append one of the following options directly and without space:
+ * B, K, KB, MB, M, GB, G, TB, T, PB, P
+ *
+ * WARNING: THIS SETTING IS SYSTEM WIDE AND COULD CAUSE DATA LOSS. USE THIS ONLY IF YOU KNOW WHAT YOU ARE DOING.
```

Comment:
```suggestion
 * Define trashbin threshold size
 * Define a threshold for files to skip the trashbin and delete immediately
 * Once the size of a resource is greater than or equal the given value, the trashbin will be skipped.
 * File sizes are valid for all mount points, take care when defining the threshold.
 *
 * All positive numbers and zero is allowed. Append one of the following options directly and without space:
 * B, K, KB, MB, M, GB, G, TB, T, PB, P
 *
```
> and zero is allowed

means, you can not have a zero length file = no file at all (but directories only) ... hmmm ... does this make sense?